spacy/tests/pipeline/test_tagger.py

```diff
@@ -136,7 +160,7 @@ def test_no_data():
 def test_incomplete_data():
     # Test that the tagger works with incomplete information
     nlp = English()
-    nlp.add_pipe("tagger")
+    nlp.add_pipe("tagger", config=dict(label_smoothing=0.0))
```

Comment:
Why exactly did you need to add this?

Reply:
Label smoothing changes the loss a bit because the logits are slightly smoother, so the test fails with `label_smoothing=0.05`. It's just a question of precision in the end, the loss is still the right order of magnitude - eg. `assert 3.28570704368758e-05 < 1e-05`.